clevertap-core/src/main/java/com/clevertap/android/sdk/Constants.java

```diff
@@ -122,6 +122,8 @@ public interface Constants {
     String INAPP_MAX_PER_DAY_KEY = "imp";
     String INAPP_WINDOW = "w";
     String INAPP_KEY = "inApp";
+    String INAPP_TEMPLATE_NAME = "templateName";
```

Comment:
can we add new constants in new feature specific package

Reply:
I will move them to the sole class that needs them